docs/udf.rst

```diff
@@ -279,8 +279,46 @@ instead of the original digital number range (thousands):
 
 Illustration of data chunking in ``apply`` with a  UDF
 ========================================================
+Chunk_polygon can be used to apply a specific kind of process to spatial chunks of a data cube. In the following example, we repeat the method of rescaling a data cube using chunk_polygon. Furthermore, in this example, we show that with the Geojson file, users can try with shapefile if they want to and fed in as a feature collection. Though a function to read JSON is shown here, you can replace it with your process.
 
-TODO
+.. code-block:: python
+    :linenos:
+    :caption: UDF alongwith chunk_polygon 
+    :emphasize-lines: 14-25
+
+    import openeo
+
+    # Create connection to openEO back-end
+    connection = openeo.connect("...").authenticate_oidc()
+
+    # Load your data cube based on your prefernce
+
+    S2_cube = eoconn.load_collection(
+        "SENTINEL2_L2A",
+        temporal_extent = ["2022-06-04", "2022-08-04"],
+        bands = ["B02", "B03", "B04"]
+    )
+
+    # Create a UDF object from inline source code.
+    my_code = """
+    from openeo.udf import XarrayDataCube
+
+    def apply_datacube(cube: XarrayDataCube, context: dict) -> XarrayDataCube:
+        array = cube.get_array()
+        array.values = 0.0001* array.values
+        return cube
+    """
+    my_udf = lambda data: data.run_udf(udf=my_code,runtime='python')
```

Comment:
I would avoid using this complicated `lambda` construct, and prefer the new `openeo.UDF`  approach as illustrated in the "A first example: apply with an UDF to rescale pixel values" section

Reply:
Actually, initially I used the openeo.UDF method, but received this error:
`AttributeError: 'NoneType' object has no attribute 'list_udf_runtimes'`

Thus I switched to this method, to avoid the error.

Reply:
I think I found the issue and fixed it in 8739413938e5fed44f76547e297511036b228b90, can you try again with that fix?

Reply:
I see from  that stack trace that you are still using an old version of the library. Are you sure you switched to latest master from git and refreshed your notebook (assuming you are working in a notebook)?

Reply:
Yes, it worked, and I have updated my UDF.